doc/CONFIGURATION.md

```diff
@@ -357,23 +357,23 @@ Access list (ACL) file format
 Some directives in the main configuration can refer to ACL files. ACLs are
 used to allow and deny connections based on the client's IP address.
 
-If an ACL is not configured for a port listener, all connections will be
-allowed. If an ACL is configured, the default is to not allow any
+If no ACL file is specified for a port listener, all connections will be
+allowed. If an ACL file is configured, the default is to not allow any
 connections unless an "allow" rule permits it.
 
-Rules in an ACL are processed sequentially, starting from the beginning. The
+Rules in an ACL file are processed sequentially, starting from the beginning. The
 first `allow` or `deny` rule matching the address of the connecting client
 is applied.
 
 The following two IPv6 lines deny the `dead:beef:f00d::/48` subnet first,
 and then allow the rest of the `dead:beef::/32` network around it. All other
-IPv6 connections are denied.
```

Comment:
I would not do this change like that. The IPv6 lines do not have any effect on IPv4.

The old wording around it kind of assumes that both the first IPv6 block and the following IPv4 block are present in the same ACL. Maybe that needs to be more explicit, for example by bringing both IPv4 and IPv6 rules in the same example block, numbering the lines and referring to them by those numbers in the description.